CONTRIBUTING.md

```diff
@@ -51,6 +51,16 @@ composer install
 
 Be sure to validate and test your code locally using the provided Composer test scripts (`composer test`) before opening a PR.
 
+### Testing the `update` command
+
+Any changes to the `acli update` command should be manually tested using the following steps:
+
+1. Replace `@package_version@` on this line with `v1.0.0-rc4` or any older version string: https://github.com/acquia/cli/blob/v1.0.0-rc5/bin/acli#L87
+2. Clear and rebuild your Symfony cache: `./bin/acli cache:clear && ./bin/acli`
+4. Install Box if you haven't already: `composer box-install`
+5. Compile phar: `composer box-compile`
```

Comment:
Keeps failing with the following message:
```
[cloudservicesdev|hosting-dev:imnet] ~/acquia/cli$ composer box-compile
> php box.phar compile

    ____            
   / __ )____  _  __
  / __  / __ \| |/_/
 / /_/ / /_/ />  <  
/_____/\____/_/|_|  
                    

Box version 3.8.4@120b0a3 2019-12-13 17:22:43 UTC

 // Loading the configuration file                                              
 // "/home/imnet/acquia/cli/box.json".                                          

 [ERROR] The configuration file is invalid.                                     


In Assertion.php line 888:
                                                                               
  "files-bin" must contain a list of existing files. Could not find "/home/im  
  net/acquia/cli/.env.local.php".                                              
                                                                               

compile [-c|--config CONFIG] [--debug] [--no-parallel] [--no-restart] [--dev] [--no-config] [--with-docker] [-d|--working-dir WORKING-DIR]

Script php box.phar compile handling the box-compile 
```

Reply:
Sorry you have to run `composer dump-env prod` to generate the file, I updated the doc